src/Button/Button.js

```diff
@@ -107,7 +178,8 @@ Button.propTypes = {
   block: PropTypes.bool,
   disabled: PropTypes.bool,
   loading: PropTypes.bool,
-  transparent: PropTypes.bool,
+  icon: PropTypes.element,
```

Comment:
I think it may be more ergonomic if this was a string so we don't have to import the Icon component every time we want to use: `<Button leftIcon="alert" leftIconProps={{ size: 48 }} />`

Reply:
@kylealwyn I updated to support this pattern. Going by your example I went ahead and added support for `leftIcon` & `rightIcon`.